### ISSUE TITLE
mediaite.com: breakage

### ISSUE THREAD

**Issue by Mayorgoldiewilson223** [uBlockOrigin/uAssets#17322]
### Prerequisites

- [x] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.mediaite.com/tv/fox-news-anchor-martha-maccallum-fed-up-after-another-mass-shooting-we-have-a-serious-problem-in-this-country/`

### Category

breakage

### Description

Video not loading

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.48.0
Chromium: 111
filterset (summary): 
  network: 87987
  cosmetic: 39678
  scriptlet: 17467
  html: 0
listset (total-discarded, last-updated): 
  default: 
    user-filters: 0-0, never
    easylist: 65003-45, now
    easyprivacy: 30741-1175, now
    plowe-0: 3751-934, now
    ublock-abuse: 78-0, now
    ublock-badware: 5438-114, now
    ublock-filters: 33093-93, now
    ublock-privacy: 294-2, now
    ublock-quick-fixes: 448-12, now
    ublock-unbreak: 1941-42, now
    urlhaus-1: 6938-0, now
filterset (user): [empty]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 2573 ms
  maxAssetCacheWait: 107 ms
popupPanel: 
  blocked: 42
  blockedDetails: 
    ads-twitter.com: 1
    adsrvr.org: 1
    adthrive.com: 18
    amazon-adsystem.com: 1
    bounceexchange.com: 1
    chartbeat.com: 1
    confiant-integrations.net: 1
    doubleclick.net: 2
    facebook.net: 1
    google-analytics.com: 3
    googlesyndication.com: 1
    ntv.io: 1
    onesignal.com: 1
    powerad.ai: 1
    profusesupport.com: 1
    quantserve.com: 1
    scorecardresearch.com: 2
    simple-cdn.s3.amazonaws.com: 1
    taboola.com: 1
    twitter.com: 1
    unpkg.com: 1
```
</details>

**Comment by Procyon-b** [comment]
Following up from the reddit post.
I'm able to reproduce the issue, but not on all attempts. I'm using an incognito to test.
- I open a new incognito window
- I load the site
- if it displays/plays the videos, it works for all of them

- I close all incognito windows and restart the test
- if videos are not shown, it's the same on all pages

I've compared cookies, local storage and session storage, and I can't see a real difference.

I tried clearing cookies and storage but it only worked once. (even before I started using an incognito window to test)
